examples/rss-feed/Readme.md

```diff
@@ -0,0 +1,179 @@
+
+	$ mkdir rss-feed-example
+
+Now that the directory is created, I'm going to open it up with
+
+	$ cd rss-feed-example
+
+##Creating and Running your package.json file 
+
+Once your in the projects directory, create a file called `package.json`. The `package.json` file will have information relative to your project and is necessary for npm to install different tools and modules that will help you while building your project. The contents of your `package.json` should look like this. 
+
+	{
+  		"name": "rss-feed-example",
+		"version": "1.0.0",
+		"dependencies": {
+		    "handlebars": "^4.0.5",
+		    "metalsmith": "^2.1.0",
+		    "metalsmith-layouts": "^1.4.1",
+		    "metalsmith-markdown": "^0.2.1",
+		    "metalsmith-permalinks": "^0.5.0",
+			"metalsmith-collections": "^0.7.0",
+			"metalsmith-feed": "^0.2.0"	
+		}
+	}
+
+Let's go through the fields in this file. 
+
+1. The name field is the name of your project and is required.
+2. The version number is the version number of your project and is required.
+3. Dependencies are the dependencies the project will use.
+4. handlebars will enable handlebars templating to be used in our project. 
+5. metalsmith allows metalsmith to be used in our project 
+6. metalsmith-layouts helps us generate templates in conjunction with handlebars. 
+7. metalsmith-markdown enables markdown files to be compiled into html files.
+8. metalsmith-permalinks will prettify our urls and nest our subdirectories to make them more managable. 
+9. metalsmith-collection helps us group different files and iterate over these files in other places in our project.
+10. metalsmit-feed will generate the rss feed for us based on our metalsmith-collection plugin options.
+
+Now that we have created our package.json, file let's run it. If npm completes the installation with no errors you should see a new folder in the directory named `node-modules`, which will have all the dependencies we listed in the `package.json` file. 
+
+	$ npm install
+
+##Creating Our index.js file
+
+Lets create a simple index.js file in our projects root directory. The file should look like this: 
+
+	var Metalsmith  = require('metalsmith');
+	var markdown    = require('metalsmith-markdown');
+	var layouts     = require('metalsmith-layouts');
+	var permalinks  = require('metalsmith-permalinks');
+	var collections = require('metalsmith-collections');
+	var feed        = require('metalsmith-feed');
+
+
+	Metalsmith(__dirname)
+	  .metadata({
+	    title: "RSS Feed Example",
+	    description: "It's about saying »Hello« to the World and letting the world keep up with your blog via RSS feed.",
+	    generator: "Metalsmith",
+	    url: "http://www.metalsmith.io/",
+	    site:{
+	      title: "RSS Feed Example",
+	      url: "http://www.metalsmith.io/",
+	      author: "George RR Martin"
+	    }
+	  })
+	  .source('./src')
+	  .destination('./build')
+	  .clean(false)
```

Comment:
In this example clean doesn't really do anything. I had been making a lot of PR's last week and copied a bunch of code for this PR. We can get rid of clean.